homeassistant/components/onboarding/views.py

```diff
@@ -94,6 +94,10 @@ class UserOnboardingView(_BaseOnboardingView):
             })
             await provider.data.async_save()
             await hass.auth.async_link_user(user, credentials)
+            if 'person' in hass.config.components:
+                await hass.components.person.async_create_person(
```

Comment:
why not `hass.data['person'].async_create_person`

Reply:
`hass.data['person']` is an anti pattern. Components should only use `@bind_hass` APIs offered by other components to interact.

We don't create persons from config UI… yet 😉 It won't need any backend changes